test/contact-summary.spec.js

```diff
@@ -115,3 +117,36 @@ describe('getContactSummary', () => {
     }
   });
 });
+
+describe('cht.v1 in contact summary ', () => {
+
+  const harness = new Harness({
+    directory: path.join(__dirname, 'collateral', 'project-with-source'),
```

Comment:
is it possible to just change [this](https://github.com/medic/cht-conf-test-harness/blob/341093bac194badd788387d3c6b0ad5dd548bfdf/test/contact-summary.spec.js#L9) to `project-with-source`? and avoid this boilerplate? 

This test class seems to mostly not care about what is in the contact-summary code

Reply:
Yes it is possible, then a change would be required [here](https://github.com/medic/cht-conf-test-harness/blob/341093bac194badd788387d3c6b0ad5dd548bfdf/test/contact-summary.spec.js#L90) because `project-with-source` does not have `contact-summary cards`  but `project-without-source` does have.